<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::TextureTab</class>
 <widget class="QWidget" name="GammaRay::TextureTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="GammaRay::TextureViewWidget" name="textureView" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="textureInfo" native="true">
     <layout class="QHBoxLayout" name="textureInfoLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="caption">
        <property name="text">
         <string>&lt;b&gt;Identified Problems&lt;/b&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textureWasteLabel">
        <property name="text">
         <string>Texture Waste:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::TextureViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">textureextension/textureviewwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
